<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSelect</class>
 <widget class="QDialog" name="LevelSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关卡选择</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Arial Unicode MS&quot;;
color: #2E8B57;</string>
     </property>
     <property name="text">
      <string>选择关卡</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Arial Unicode MS&quot;;
background-color: #f0f0f0;
border: 2px solid #cccccc;
border-radius: 8px;
padding: 8px;</string>
       </property>
       <property name="text">
        <string>返回主菜单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResetProgress">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Arial Unicode MS&quot;;
background-color: #ffcccb;
border: 2px solid #ff6347;
border-radius: 8px;
padding: 8px;</string>
       </property>
       <property name="text">
        <string>重置进度</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>